Servlets/src/main/java/ua/goit/servlets/servlets/Projects.java

```diff
@@ -0,0 +1,46 @@
+package ua.goit.servlets.servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import ua.goit.servlets.dao.CategoriesList;
+import ua.goit.servlets.model.Category;
+import ua.goit.servlets.model.Project;
+
+public class Projects extends HttpServlet {
+	static final long serialVersionUID = 15909L;
+	List<Category> categories;
+
+	public void init() throws ServletException {
+		super.init();
+		categories = CategoriesList.newCategoryInstance();
+	}
+
+	@Override
+	public void doGet(HttpServletRequest req, HttpServletResponse res)throws IOException, ServletException {
+		res.setContentType("text/html");
+		PrintWriter writer = res.getWriter();
+		
+		String categoryName = req.getParameter("category");
+		Category category;
```

Comment:
Please don't create the local var in advance. It's used in a single place and creating it in advance makes me confused